plugins/azure-devops-backend/src/api/AzureDevOpsApi.test.ts

```diff
@@ -16,17 +16,21 @@
 
 import {
   Build,
+  DefinitionReference,
+} from 'azure-devops-node-api/interfaces/BuildInterfaces';
+import {
   BuildResult,
   BuildStatus,
-  GitPullRequest,
-  GitRepository,
   PullRequest,
   PullRequestStatus,
   RepoBuild,
-} from './types';
+} from '@backstage/plugin-azure-devops-common';
+import {
+  GitPullRequest,
```

Comment:
Do we still want to import these from the `node-api` or should we be importing it from `plugin-azure-devops-common`?

Reply:
Should be from `@backstage/plugin-azure-devops-common`.